ymir/backend/src/common/common_utils/labels.py

```diff
@@ -56,128 +59,154 @@ def _check_labels(cls, labels: List[SingleLabel]) -> List[SingleLabel]:
         return labels
 
 
-def labels_file_name() -> str:
-    return 'labels.yaml'
+class UserLabels(LabelStorage):
+    id_to_name: Dict[int, str] = {}
+    name_to_id: Dict[str, int] = {}
+    name_aliases_to_id: Dict[str, int] = {}
+
+    def __init__(self, **data: Any):
```

Comment:
use [root_validator](https://pydantic-docs.helpmanual.io/usage/validators/#root-validators).

for example:

```python
     @root_validator()
     def make_up_fields(cls, values: Any):
         id_to_name = {}
         name_to_id = {}
         name_aliases_to_id = {}
         for label in values["labels"]:
             id_to_name[label.id] = label.name
             name_to_id[label.name] = label.id
             name_aliases_to_id[label.name] = label.id
             for alias in label.aliases:
                 name_aliases_to_id[alias] = label.id
         values["id_to_name"] = id_to_name
         values["name_to_id"] = name_to_id
         values["name_aliases_to_id"] = name_aliases_to_id
         return values
```

Reply:
done